Templates/SasToken-Node/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "defaultFunctionName": "SasTokenNodeJS",
+    "description": "A Node.js function that generates a SAS token for Azure Storage for a given container and blob name",
+    "name": "SAS Token Generator - Node",
+    "language": "JavaScript",
+    "category": [
+        "Samples",
+        "API & Webhooks"
```

Comment:
This should only be in Samples, not in the core API & WebHooks, since it is a pretty specialized sample scenario. For all our other existing samples, I don't think we cross categorize any of them like this?